_haikus/struggling.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Struggling
+author: Ashley Weston
+---
+
+What am I doing <br>
+This is confusing <br>
```

Comment:
This line should be 7 syllables and is currently 5, I would suggest changing the line to read:

```
This is very confusing <br>
```